### DISCUSSION TITLE
matching partially stripped star

### DISCUSSION THREAD

**Discussion by ZepeiX** [POSYDON-code/POSYDON#721] [Ideas]
Currently, detached step cannot find good matches to partially stripped stars. We decided to construct single partially stripped star grids.

**Comment by astroJeff** [comment]
For us to proceed, it would be useful to know how good or bad the matches to partially stripped stars are. @ZepeiX do you have a population saved that you can load up and provide some statistics on the quality of matching to partially stripped stars?
